examples/005_histogram_fit/histogram_fit.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+"""
+kafe2 example: Histogram Fit
+=======================
+
+kafe2 is not only capable of performing XY-Fits. This example shows how to handle a one dimensional dataset.
+The distribution of a random stochastic variable follows a probability density function. This density needs to be
+normalized. The fit will determine the parameters of that density function, which the dataset is most likely to follow.
+If the given function is not normalized, please make sure to add an independent scale factor. The hist fit will try to
+normalize the function to one for the bin range.
```

Comment:
Even in the case that you add a scale factor?

Reply:
This is a subtle, but important point. The HistFit makes the assumption that the density supplied by users is normalized to 1.0 over the entire histogram range. Before computing the `chi2`, the density is scaled up by a factor of *N*, which is the number of entries in the histogram, so the overall normalization of the model function is fixed to be exactly *N* all the time.

If this is not desired, a "scale factor" *c* (usually called a *normalization parameter*) can be multiplied to the original density, meaning that the model function will be normalized to *Nc*. The difference is that now *c* is a free fit parameter, so the normalization is not fixed but *inferred from the data*.

While both use-cases are valid, I feel we should focus on only one of the here, since this is a tutorial for beginners. Saying "please make sure to add an independent scale factor" sound a bit cryptic and beginners probably wouldn't know what you are referring to.

Reply:
Do you suggest to remove the mention of a nomalization parameter?
Do you have any suggestions on how to word this?

Reply:
I would remove the mention here. We can address the topic in the upcoming advanced user guide.

Reply:
@dsavoiu this is the histogram docstring I mentioned. You marked it as resolved. I didn't change anything here.